tidb-cloud/integrate-tidbcloud-with-aws-lambda.md

```diff
@@ -0,0 +1,196 @@
+            ```
+
+        3. Install the dependencies:
+
+            ```shell
+            yarn
+            ```
+
+    2. Rebuild the code bundle.
+
+        1. Build the code bundle.
+
+            ```shell
+            yarn build
+            ```
+
+        2. Check the `aws-lambda-cloudformation/dist/index.zip` file.
+        3. Right-click the `index.zip` file and select **Download**.
+
+4. Upload the rebuilt code bundle to your own S3 bucket.
+
+    1. Visit the [S3 service](https://console.aws.amazon.com/s3) in the AWS Management Console.
+    2. Create a new bucket in your selected region.
+    3. Upload the `index.zip` file to the bucket.
+    4. Note down the S3 bucket name and region for later use.
+
+</details>
+
+## Step 1: Set up the project using AWS CloudFormation
+
+To set up the bookshop project using AWS CloudFormation, do the following:
+
+1. Navigate to the AWS Management Console and access the [AWS CloudFormation service](https://console.aws.amazon.com/cloudformation).
+2. Click **Create Stack** > **With new resources (standard)**.
+3. On the **Create Stack** settings page, complete the stack creation process.
+
+    1. In the **Prerequisite** panel, select **Template is ready**.
+    2. Upload the template file (either YAML or JSON), and click **Next**.
+
+        If you do not have the file yet, download it from [GitHub](https://github.com/pingcap/TiDB-Lambda-integration/releases/latest). The file contains the AWS CloudFormation template that creates the necessary resources for the project.
+
+        ![aws-lambda-cf-create-stack](/media/develop/aws-lambda-cf-create-stack.png)
+
+    3. Specify stack details.
+
+        - If you use `us-east-1` as your cluster region, fill in the fields as in the following screenshot:
+
+            ![aws-lambda-cf-stack-details](/media/develop/aws-lambda-cf-stack-config.png)
+
+            - **Stack name**: enter the stack name.
+            - **S3Bucket**: enter the S3 bucket where you store the zip file.
+            - **S3Key**: enter the S3 key.
+            - **TiDBDatabase**: enter the TiDB Cloud cluster name.
+            - **TiDBHost**: enter the host URL for TiDB Cloud database access. Enter `localhost`.
+            - **TiDBPassword**: enter the password for TiDB Cloud database access.
+            - **TiDBPort**: enter the port for TiDB Cloud database access.
+            - **TiDBUser**: enter the user name for TiDB Cloud database access.
+
+        - If you use a different AWS region other than `us-east-1`, follow these steps:
+
+            1. Refer to [Modify and rebuild the Lambda function code if necessary](#prerequisites) to modify the Lambda function code, rebuild and upload the code bundle to your own S3 bucket.
+            2. In the stack details fields, specify the S3 bucket name and region in the `S3Bucket` and `S3Key` parameters according to your own configuration.
+            3. Fill in other fields as in the preceding screenshot.
+
```

Comment:
After filling in the fields for stack details, there is one more page for users to configure stack option, and another page for "Review and create". 

![image](https://github.com/user-attachments/assets/cf5c65ac-69c8-4118-922f-3462b1391ad9)

Reply:
Added